tests/test_project.py

```diff
@@ -260,6 +294,45 @@ async def test_modify_environment_project(client_v2):
     assert result.code == 200
 
 
+async def test_delete_environment_project_v2(client_v2):
+    """Test modifying the project of an environment"""
```

Comment:
Is this test exactly the same as the one above but with 2 environments instead of one ? Do we need both tests, maybe this one is enough ?

Reply:
I tend to agree, I wanted to cover the V1 endpoint (even if it relies on the same logic as V2)

Reply:
I see, I completely missed the fact that they were using different endpoints. Maybe we should err on the side of caution and test both versions after all, in case we update this in the future. @arnaudsjs  wdyt ?